plugins/csv-export/src/csv.ts

```diff
@@ -0,0 +1,126 @@
+import type { Collection, CollectionField, CollectionItem } from "framer-plugin"
+
+import { isFileAsset, isImageAsset } from "framer-plugin"
+import { assertNever } from "framer-plugin/utils"
```

Comment:
We are not exporting this from the package so you have to define it inside the plugin itself